src/services/queries/Providers.js

```diff
@@ -0,0 +1,30 @@
+import gql from 'graphql-tag';
+
+export const getListOfProvidersQuery = gql`
+  query getListOfProviders(
+    $query: String = "%%"
+    $limit: Int = 10
+    $offset: Int = 0
+    $order_by_name: order_by = null
```

Comment:
maybe make this renaming thing outside the query, please, as the query is agnostic of such changes (business/code logic)

Reply:
Can you explain a little more please?

Reply:
```suggestion
    $order_by = null
```

You are renaming here, but I think it make it harder to maintain, you have to remember to change the query as well as the JS code

Reply:
I'll check that possibility, thanks for the suggestions!

Reply:
I will work in that

Reply:
I've fixed it now! 

Now you should pass a object with 

```{
column: 'name',
content: 'asc'
}```

for example, then column will be the column that will be used to sort the object on the query. Now we don't need to add a param for each one column, and the frontend can add more columns easily